docs/news/posts/2024-10-04-data-access.md

```diff
@@ -0,0 +1,283 @@
+the following example: Assume you wanted to obtain all integers with `pk`s in a range between 1 and 10 (both excluded)
+from a `Group` called "integers". You'd have to construct the rather convoluted query as shown in the left column of the
+following table:
+
+<!-- prettier-ignore -->
+<!-- Taken from: https://gist.github.com/panoply/176101828af8393adc821e49578ac588 -->
+<table style="table-layout: fixed; width: 100%">
+  <thead>
+    <tr>
+      <th width="500px"> Old `QueryBuilder` syntax </th>
+      <th width="500px"> New `QueryBuilder` syntax </th>
+    </tr>
+  </thead>
+  <tbody>
+  <tr width="600px">
+      <td style="word-break:break-all;">
+
+```python
+QueryBuilder().append(
+    Group,
+    filters={
+        "label": "integers",
+    },
+    project=["label"],
+    tag="group",
+).append(
+    Int,
+    with_incoming="group",
+    filters={
+        "and": [
+            {"pk": {">": 1}},
+            {"pk": {"<": 10}},
+        ]
+    },
+    project=["pk", "attributes.value"],
+)
+```
+
+</td>
+<td>
+
+```python
+QueryBuilder().append(
+    Group,
+    filters=Group.fields.label == "integers",
+    project=[Group.fields.label],
+    tag="group",
+).append(
+    Int,
+    with_incoming="group",
+    filters=(Int.fields.pk > 1) & (Int.fields.pk < 10),
+    project=[Int.fields.pk, Int.fields.value],
+)
+
+
+
+
+
+
+
+```
+
+</td>
+</tr>
+
+  </tbody>
+</table>
+
+With the new syntax, `Node` attributes are directly accessible via
+`.fields` and logic can be applied to them directly. Through this, for example the filter on the values of
+the `pk`s reduces from:
+
+```python
+filters={
+    "and": [
+        {"pk": {">": 1}},
+        {"pk": {"<": 10}},
+    ]
+},
+```
+
+to the more concise:
+
+```python
+filters=(Int.fields.pk > 1) & (Int.fields.pk < 10),
+```
+
+in which the `"and"` condition can be expressed via `&` and applied on the relevant entities. Furthermore,
+accessing through the `.fields` attribute, e.g. in the updated `project`ion:
+
+```python
+project=[Int.fields.pk, Int.fields.value]
+```
+
+due is less error-prone than the string access in the provious version
+
+```python
+project=["pk", "attributes.value"]
+```
+
+thanks to autocompletion.
+
+Feel free to explore the new syntax (any feedback welcome), and query away!
+
+***
+
+## Relevant PRs
+
+For the more tech-savy among us, here are the relevant PRs of the changes outlined in this blog post:
+
+- [[#6276]](https://github.com/aiidateam/aiida-core/pull/6023) Add CLI command to dump inputs/outputs of `CalcJob`/`WorkChain`
+- [[#6245]](https://github.com/aiidateam/aiida-core/pull/6245) ✨ NEW: Add `orm.Entity.fields` interface for `QueryBuilder`
+  (cont.) [and linked PRs]
+
+## Footnotes
+
+[^1]:
+  The file repository is based on the [`disk-objectstore`](https://github.com/aiidateam/disk-objectstore)
+  implementation. If you ever wondered what the `_dos` appendix of the `core.psql_dos` and `core.sqlite_dos` storage
+  backends means, now you know! 😉
+
```

Comment:
Footnotes do not work properly. Maybe the linebreak?

Reply:
Hm, weird, they worked fine in the other blog post despite the linebreaks. Will have a look!

Reply:
Was the indent that was wrong...